### PR TITLE
MT-86437 Updated project settings, fixed compiler warnings

### PR DESCRIPTION
**Pull request by adamjansch**
[MT-86437]

### CODE REVIEW

File: COSTouchVisualizer.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
```
Comment (leamerluck) [inline COSTouchVisualizer.podspec:1]: Did you verify this change should be made?

Comment (adamjansch) [reply COSTouchVisualizer.podspec:1]: Which change in particular?

Comment (leamerluck) [reply COSTouchVisualizer.podspec:1]: Setting the version an, homepage, and source?

Comment (adamjansch) [reply COSTouchVisualizer.podspec:1]: I think incrementing the version makes sense, as then we know that we're using our new version.

As for the other two, it feels like the right way to go. I don't think this file has a huge amount of bearing on anything at the moment – I think it's really for populating info on https://cocoapods.org/pods, but we're not submitting our repos there as far as I understand.

@multitracks Have you got a particular way you want us to go on this?

File: Classes/COSTouchVisualizer.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -209,10 +211,14 @@
...
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+					"@loader_path/Frameworks",
+				);
 				MTL_ENABLE_DEBUG_INFO = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = com.conopsys.TouchVisualizer;
+				PRODUCT_BUNDLE_IDENTIFIER = com.multitracks.TouchVisualizer;
```
Comment (adamjansch) [inline Classes/COSTouchVisualizer.xcodeproj/project.pbxproj:221]: @multitracks Should we be updating the bundle identifier of these forked repos to reference multitracks.com?

File: COSTouchVisualizer.podspec (hunk 1 above)
Comment (multitracks) [reply COSTouchVisualizer.podspec:1]: This is great 👍

File: Classes/COSTouchVisualizer.xcodeproj/project.pbxproj (hunk 2 above)
Comment (multitracks) [reply Classes/COSTouchVisualizer.xcodeproj/project.pbxproj:221]: Sounds good 👍

**Pull request merged**